pkg/vpcmodel/nodesConnectivity.go

```diff
@@ -61,6 +61,11 @@ func (c *VPCConfig) GetVPCNetworkConnectivity(grouping, lbAbstraction bool) (res
 	res.computeAllowedConnsCombined()
 	res.computeAllowedStatefulConnections()
 	if lbAbstraction {
+		// load balancer abstraction:
+		// currently, AllowedConnsCombined contains the private IPs of the load balancer.
+		// the abstraction creates new AllowedConnsCombined,
+		// it replaces the private IPs in the with the load balancer itself
+		// see details at nodeSetConnectivityAbstraction()
 		for _, lb := range c.LoadBalancers {
 			res.AllowedConnsCombined = nodeSetConnectivityAbstraction(res.AllowedConnsCombined, lb)
 		}
```

Comment:
can change the call to a main function that gets `AllowedConnsCombined` without any abstraction and the set of all load balancers, and returns the final result of the abstraction? (instead of iterating the LBs here)